docs/contributing.md

```diff
@@ -100,6 +100,13 @@ You can also run a single test script like this:
 $ nox -s test -- tests/test_multipart.py
 ```
 
+!!! tip
+    `nox` is configured to [reuse existing virtual environments](https://nox.thea.codes/en/stable/usage.html#re-using-virtualenvs). To force tests to run in a clean virtual environment (e.g. because dependencies have changed), use:
```

Comment:
We can add `--upgrade` to `session.install()` steps to ensure we're getting latest dependencies even in resused venvs.

Reply:
Good idea, yup. I pushed a commit with this change. :)